test_version.py

```diff
@@ -0,0 +1,13 @@
+"""Check whether the __version__ attribute is set correctly."""
```

Comment:
Nitpick: Can we move this into a separate `tests/` directory?

For this, there shouldn't be an import issue and I personally prefer to keep the root dir as clean as possible. If there is a reason to keep it here, I don't mind though.

Reply:
SGTM, done.